### ISSUE TITLE
Accessibility][On Focus - Settings-Publish-Create Azure Resource]: On Activating 'Next' button without providing any values on all fields, focus is not landing to first field.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2109]
**User Experience:**
Keyboard dependent users would face difficulty if focus does not visible anywhere on the page/dialog that opens up after selecting any control.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Select 'Settings' from left navigation.
5. Select 'Publish' button available under 'Configuration'.
6. Navigate to "Create azure resources" and activate it.
7. "Set bot name and password" dialog gets opened.
8. Leave all the edit field empty and press enter on 'Next' button and observe the issue.

**Actual Result:**
On Activating 'Next' button without providing any values on all fields, focus is not landing to first mandatory field.

**Note:**
Focus is always landing on 'Azure Region' edit field when 'Next' button is activated without providing any values on all fields.

**Expected Result:**
On Activating 'Next' button without providing any values on all fields, focus should land to first mandatory field.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}

**Attachment:**
[MAS3.2.1_Settings_Publish_Create Azure.zip](https://github.com/microsoft/BotFramework-Composer/files/4261307/MAS3.2.1_Settings_Publish_Create.Azure.zip)
![MAS3 2 1_Settngs_Publish](https://user-images.githubusercontent.com/60919065/75652951-6fd5ae00-5c82-11ea-99cb-58ec8d281f4f.png)

**Comment by corinagum** [comment]
Request for UX improvement:
![image](https://user-images.githubusercontent.com/14900841/76466016-07827d00-63a4-11ea-881e-89d6bbbc7d0a.png)

Spoke with @DesignPolice, summarized below:
- In Emulator, dialogs with unfilled required fields disable the 'Next' button until they are filled. 
- Here, user isn't informed of errors until after clicking Next. 
- DesignPolice and I agree that behavior should change to match Emulator - pressing 'Next' is not necessary to know that required fields are empty.
- Currently I'm not sure of the T-shirt size to do this throughout the app. Could @a-b-r-o-w-n or @cwhitten weigh in? 
- If too large, I'm assuming we would want to do the short-and-dirty, keep behavior but return focus to first empty required field?

**Comment by corinagum** [comment]
Offline discussion summary: disable 'Next' button route looks good.

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.
**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
